api/example.env

```diff
@@ -1,5 +1,7 @@
 API_PORT=3001
 
+DATABASE_URL=postgresql://reach4help_api:reach4help@localhost:15432/reach4help_api_db
```

Comment:
This env var is used only on Heroku. For local dev this will be undifined and TYPEORM_* will be used instead. You can check */database/ormconfig.ts

Reply:
hmm I needed it initially, but now it seems to work without it. I think adding dotenv.config() must have fixed it